dist/qs.js

```diff
@@ -329,6 +329,18 @@ var arrayPrefixGenerators = {
     }
 };
 
+var objectPrefixGenerators = {
```

Comment:
please revert all changes to this file; it's only updated at release time.

Reply:
Done